source/objects/Primitives.h

```diff
@@ -75,9 +77,14 @@ struct TriangleBackfaceCull
 		const bvh::Vector3<Scalar> p1,
 		const bvh::Vector3<Scalar> p2,
 		const int16_t material,
+		const glm::vec2 uvs[3],
 		const bool oneSided = false
-	) : p0(p0), e1(p0 - p1), e2(p2 - p0), material(material), oneSided(oneSided)
+	) : p0(p0), e1(p0 - p1), e2(p2 - p0), material(material), uvs(), oneSided(oneSided)
```

Comment:
is this empty initialisation doing anything?

Reply:
No